server/routes/auth/providers/oidc.ts

```diff
@@ -85,10 +90,13 @@ if (env.OIDC_CLIENT_ID && env.OIDC_CLIENT_SECRET) {
             throw OIDCMalformedUserInfoError();
           }
 
-          const subdomain = domain.split(".")[0];
+          // remove the TLD and form a subdomain from the remaining
+          const subdomain = domain.split(".").slice(0, -1).join("-");
```

Comment:
Wanna move this to a utils method too? Seems like it changed in a couple of spots